src/instrumentsscene/qml/MuseScore/InstrumentsScene/InstrumentsPanel.qml

```diff
@@ -117,7 +120,7 @@ Item {
             Layout.rightMargin: 20
 
             //: Text between %1 and %2 will be printed in a bold font.
-            text: qsTrc("instruments", "There are no instruments in your score. To choose some, press %1Add%2, or use the shortcut %1‘i’%2")
+            text: qsTrc("instruments", "There are no instruments in your score. Add at least one instrument.")
                   .arg("<b>").arg("</b>")
```

Comment:
You can delete this line. The `.arg()` replaces the `%1` with string passed as its parameter (so `<b>`)

Reply:
Thanks, spotted